frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -259,7 +259,7 @@ export const SidePanelSupport = (): JSX.Element => {
                                         targetBlank
                                         className="mt-2"
                                     >
-                                        Email an engineer
+                                        Email support
```

Comment:
```suggestion
                                        Email our support engineers
```

I've proposed this before, but I think this keeps the best of both worlds.